HealthcareSystem/PatientWebApp/DTOs/PharmacyDTO.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace PatientWebApp.DTOs
+{
+    public class PharmacyDTO
+    {
+        public int Id { get; set; }
+        public string Name { get; set; }
+        public string ApiKey { get; set; }
+        public string Url { get; set; }
+        public string ActionsBenefitsExchangeName { get; set; }
+        public bool ActionsBenefitsSubscribed { get; set; } = false;
+        public string GrpcHost { get; set; }
+        public int GrpcPort { get; set; }
```

Comment:
Why would you expose any of these to these in the PatientWebApp API? Why not just put the Pharmacy name in the ActionBenefitDTO instead of the Id and put it together when getting them?

Reply:
Fixed in a5ad64709381a12cb71e9eaa5b25cbba14c76a01.